recipes/crlibm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "crlibm" %}
+{% set version = "1.0b5" %}
+{% set version_commit = "eb30637" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/taschini/crlibm/tarball/{{ version_commit }}
+  fn: crlibm-{{ version_commit }}.tar.gz
+  sha256: 2c0b6da165a1f34e99da901a4d7ef59f6b81450bf8649c14581a7cb7e8ffe14a
+  folder: src
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - automake
+    - make
+
+test:
+  commands:
+    - "test -f $PREFIX/lib/libcrlibm.a"
```

Comment:
Is there a way to make this a shared library?

Reply:
Doesn't look like that is possible.